bundles/org.openhab.transform.regex/README.md

```diff
@@ -102,7 +104,7 @@ Please note: This profile is a one-way transformation, i.e. only values from a d
 
 * A full [introduction](https://www.w3schools.com/jsref/jsref_obj_regexp.asp) for regular expression is available at W3School.
 * Online validator help to check the syntax of a regex and give information how to design it.
-    * [Regex 101](https://regex101.com/)
-    * [Regex R](https://regexr.com/)
-    * [ExtendsClass](https://extendsclass.com/regex-tester.html)
-    * [Softwium](https://softwium.com/regex-explainer/)
+  * [Regex 101](https://regex101.com/)
+  * [Regex R](https://regexr.com/)
+  * [ExtendsClass](https://extendsclass.com/regex-tester.html)
+  * [Softwium](https://softwium.com/regex-explainer/)
```

Comment:
I think this might have been missed in #14371: can you also change bullets like this (also above):
```suggestion
  - [Regex 101](https://regex101.com/)
  - [Regex R](https://regexr.com/)
  - [ExtendsClass](https://extendsclass.com/regex-tester.html)
  - [Softwium](https://softwium.com/regex-explainer/)
```